internal/pkg/scaffold/crd.go

```diff
@@ -105,8 +101,12 @@ func (s *CRD) CustomRender() ([]byte, error) {
 
 		// controller-tools generates crd file names with no _crd.yaml suffix:
 		// <group>_<version>_<kind>.yaml.
-		path := strings.Replace(filepath.Base(i.Path), "_crd.yaml", ".yaml", 1)
-		b, err := afero.ReadFile(fs, path)
+		genPath := fmt.Sprintf("%s_%s_%s.yaml",
+			s.Resource.GoImportGroup,
+			s.Resource.Version,
+			s.Resource.LowerKind,
```

Comment:
I could totally be missing something, but is this the correct path?

I'm looking at this: https://github.com/kubernetes-sigs/controller-tools/blob/46695abfb89bb6fcd972edf4f925915434726008/pkg/crd/gen.go#L77

Reply:
It is, but this PR doesn't bump c-t, just the CRD file name change.